### PR TITLE
Add goldens for bar charts

### PR DESCRIPTION
**Pull request by abdushakoor12**
Goldens for all bar charts

### CODE REVIEW

**Comment by abdushakoor12** [comment]
Rest of bar charts in progress. Made PR a little early. Gonna finish them soom

**Comment by MelbourneDeveloper** [comment]
@abdushakoor12 thankyou so much!

I had to merge this into a temporary branch because I need to run the GitHub action for the goldens on Linux so they always match, but this is great work. Exactly what I was hoping for.

File: charts_flutter/example/lib/bar_chart/bar_gallery.dart [hunk 1]
```diff
@@ -39,6 +39,25 @@ import 'vertical_bar_label.dart';
...
+const horizontalBarLabelCustomChart =
+    'Horizontal Bar Chart with Custom Bar Labels';
+const verticalBarLabelChart = 'Vertical Bar Chart with Bar Labels';
+const sparkBarChart = 'Spark Bar Chart';
+const groupedFillColorChart = 'Grouped Fill Color Bar Chart';
+const stackedFillColorChart = 'Stacked Fill Color Bar Chart';
+const patternForwardHatchChart = 'Pattern Forward Hatch Bar Chart';
+const horizontalPatternForwardHatchChart =
+    'Horizontal Pattern Forward Hatch Bar Chart';
+const weightedPatternChart = 'Weighted Pattern Bar Chart';
```
Comment (MelbourneDeveloper) [inline charts_flutter/example/lib/bar_chart/bar_gallery.dart:59]: @abdushakoor12 Goldens FTW!

When I generated the goldens there was a slight difference for the menu golden. I couldn;t figure out what it was, so I got Claude to analyze the difference between the two files. It found that this text should be: `Weight Pattern Bar Chart` and not `Weighted Pattern Bar Chart`

image

File: charts_flutter/example/lib/bar_chart/bar_gallery.dart [hunk 2]
```diff
@@ -68,108 +87,138 @@ List<GalleryScaffold> buildGallery() {
...
+          ? VerticalBarLabelChart.withRandomData()
+          : VerticalBarLabelChart.withSampleData(),
     ),
     new GalleryScaffold(
       listTileIcon: new Icon(Icons.insert_chart),
-      title: 'Spark Bar Chart',
-      subtitle: 'Spark Bar Chart',
-      childBuilder: () => new SparkBar.withRandomData(),
+      title: sparkBarChart,
+      subtitle: 'Spark Bar Charts',
```
Comment (MelbourneDeveloper) [inline charts_flutter/example/lib/bar_chart/bar_gallery.dart:169]: @abdushakoor12 another slight change here

image

File: charts_flutter/example/lib/bar_chart/bar_gallery.dart (hunk 1 above)
Comment (MelbourneDeveloper) [reply charts_flutter/example/lib/bar_chart/bar_gallery.dart:59]: @abdushakoor12 it turns out that there were more. I fixed it all up in this [PR](https://github.com/Nimblesite/nimble_charts/pull/15/files)

Comment (abdushakoor12) [reply charts_flutter/example/lib/bar_chart/bar_gallery.dart:59]: @MelbourneDeveloper  Oh nice, like what you did with tileFinder predicate.

**Pull request merged**